facturacion_mexico/cfdi_recibidos/services/item_resolver.py

```
@@ -0,0 +1,149 @@
+		"""
+		result = self._try_mapeado(company, supplier_rfc, sat_product_key)
+		if result:
+			return result
+
+		result = self._try_especifico(supplier, no_identificacion)
+		if result:
+			return result
+
+		return {"item_code": None, "item_resolution": None}
+
+	def search_candidates(self, description: str, sat_product_key: str, company: str) -> list[dict]:
```

Ruff (0.15.14)

[warning] Unused method argument: `sat_product_key`

(ARG002)